server/apps/main/templates/main/experiences_page.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>AutSPACEs Experiences Page</title>
+    <style>
+        body {
+            background: #E8F3FC;
+        }
+        .my-custom-card{
+            background:#C4C4C4;
+			height: 699px;
+
+        }
+        .goals{
+            background:#EBF1F5;
+        
+        }
+        .navbar{
+            background: rgba(149, 239, 234, 0.42);
+			font-size: 24px;
+
+        }
+		form{
+            text-align: center;
+            font-size: x-large;
+        }
+		.mysmallCcards{
+			background: #00B2FF;
+			
+		}
+
+        
+    </style>
+</head>
+
+<body>
+	<nav class="navbar">
+		<a class="navbar-brand" href="#" </a>
+
+			 <a style="color:#000000;"href="#about">About</a>      
+			 <a style="color:#000000;"href="#share">Share</a>      
+			 <a style="color:#000000;"href="#login">Login</a>

```

Comment:
Seems like you're missing a few other buttons, @sowmya9507 . 

Could you add the rest of the buttons as defined on issue #105 ?

Reply:
Sure, will do :)